src/context/index.tsx

```diff
@@ -14,13 +14,21 @@ interface Props {
 export const AppContextProvider = ({ children }: Props) => {
   const [darkmode, setDarkmode] = useState<boolean>(false);
 
-  const contextValues = { darkmode, setDarkmode };
-  return <AppContext.Provider value={contextValues}>{children}</AppContext.Provider>;
+  const toggleDarkmode = () => {
```

Comment:
I don't mind using a toggle function, but you should pass that in as the type to the context below. 
Right now, anyone using this will see "setDarkmode" which takes in a boolean. 
A boolean that doesn't exist with the toggleDarkmode.

Reply:
oh okayyy